src/hammer-vlsi/defaults.yml

```
@@ -83,6 +83,35 @@ vlsi.core:
   # Maximum threads to use in a CAD tool invocation.
   max_threads: 1
 
+#----------------------------------------------------------------------------
+# JOB SUBMISSION
+#----------------------------------------------------------------------------
+vlsi.submit:
+  # - The submit command to use. "none", "local", or null will run on the 
+  #   current host. See hammer_submit_command.py for other options.
+  command: "local"
+  # type: List[Dict[str, Dict[str, Any]]]
+  # The list substitutes settings in order of appearance, and the first 
+  # Dict key is the command. The second dict key is the name of that 
+  # command's setting, followed by whatever type it takes.
+  settings: []
+
+  # how many output lines to buffer in memory. "" for unlimited
+  max_output_lines: null
+  # how many error lines to buffer in memory. "" for unlimited
+  max_error_lines: null
```

This is meant to be a suggestion for all 4 lines but I think GitHub doesn't know how to do it for all 4 lines...

```suggestion
  # How many output lines to buffer in memory.
  # If unset, buffer unlimited lines.
  # type: Optional[int]
  max_output_lines: null
  # How many error lines to buffer in memory.
  # If unset, buffer unlimited lines.
  # type: Optional[int]
  max_error_lines: null
```